### PR TITLE
Implement ForbiddenComment Rule.

### LINKED ISSUE #182: Rule TodoComment
Basically flags the usage of `// TODO: something`

Inspired by [this checkstyle rule](http://checkstyle.sourceforge.net/config_misc.html#TodoComment).

Sidenote: I'd not flag `// TODO something` as sometimes you can't do something regarding a Todo - [like here](https://github.com/arturbosch/detekt/blob/d8456af3e639947b773cff0516e0866825c74d2b/detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/extensions/DetektExtensionTest.kt#L40). Checkstyle also has this behavior.

### PR DESCRIPTION
**Pull request by vanniktech**
Fixes #182

### CODE REVIEW

File: detekt-cli/src/main/resources/default-detekt-config.yml [hunk 1]
```diff
@@ -115,6 +115,9 @@ style:
   active: true
   NewLineAtEndOfFile:
     active: true
+  ForbiddenComment:
+    active: true
+    values: 'TODO:,FIXME:,STOPSHIP:'
```
Comment (Mauin) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: Do we want to leave this in the default config if these are the default parameters already?

Comment (vanniktech) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: I guess so. Don't know what the standard way of this is.

Comment (Mauin) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: Me neither, but we should agree on something. Both `LateinitUsage` and `WildcardImport` are not specifying their (by default empty) exclude parameters. But we could also add them.

Comment (vanniktech) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: I think this will be resolved once #189 is implemented.

**Comment by arturbosch** [comment]
Looks good :+1: , should we rename `values`? Something like `startsWithPattern` ?

**Comment by vanniktech** [comment]
It does not need to start with it. And it's also not a pattern (regex). I'd propose something with contains

**Comment by arturbosch** [comment]
`containsString` or just `contains`? Well then values founds even better xD

**Comment by vanniktech** [comment]
Yeah I just thought of `values`. I'd say leave it and stabilize the names before 1.0.0

**Comment by vanniktech** [comment]
Cool, I added it to #109

**Pull request merged**